meetings/2020-12-15.rst

```diff
@@ -0,0 +1,76 @@
+=========================
+Meeting December 15, 2020
+=========================
+
+- With James Hester present, the discussion naturally started on features of the CIF standard and thoughts on how our effort relates to the ontology-related work of the IUCr.
+  A context for this was a previous discussion on how an ontology for materials databases relates to / should be positioned to the work of IUCr and the crystallography domain of the EMMO.
+
+  * Discussion on the degree to which relationships are/can be represented in the CIF format (and thus the implicit ontology of that standard)
+  
+    - The CIF format is, at its core, "a dictionary" with carefully defines entities that can have values (single values, tables, etc.) 
+      
+    - The definitions of these data values describe relations between them, but there is no explicit machine-readable representation of relations.
```

Comment:
Well this is not what I understood.
This is why I asked James about the "Loop a from .." and "With c as ...", which seems to be a syntactical language? dRel or DDL or something?

Reply:
So, I probably missed something there. Were these features of cif2? @sauliusg can you help out here? Is my summary point right above this incorrect? What should be changed/added to better describe to what degree cif/cif2 supports machine-readable relations between entities?

Reply:
The CIF standards consist of 2 syntaxes (CIF1 and CIF2), ontology languages (DDL2,DDLm and the older DDL1, which I didn't explicitly mention, but DDLm was the language that the definitions I showed were written in), and a series of what we call "dictionaries" which are ontologies written in DDLm or DDL2 for particular subject areas - we have the "core", "modulated structures", "magnetic", "powder" and so forth. I showed bits of the "core" dictionary. While the data names defined in the dictionaries traditionally refer to data names that appear in the CIF formats, there is no reason that they can't be linked to items expressed in other formats.

The embedded algorithms were written in one more standard called "dREL", which is orthogonal to the other standards in that these algorithms appear simply as a text string in the dictionaries. dREL could be used in any relational environment, and is an optional addition to a definition.

So I would change the summary point to (sorry, don't know how to put a diff into the comment yet):

- CIF standards include "dictionaries" that carefully define entities ("data names") that can have one or more values when appearing in data files.  Sufficient information is provided in the dictionary to form a relational schema from the data names, where these data names are the columns appearing in the relations. 
- The definitions of these data names describe only those relationships found in the relational model: "key" data names and "foreign keys".  The particular meaning underlying the foreign key relationship is not expressed.

Reply:
dREL expresses mathematical relationships between data names rather than logical ones like "is a" "is part of" "is a type of".